salt/top.sls

```diff
@@ -23,6 +23,8 @@ base:
   'roles:kube-master':
     - match: grain
     - kubernetes-master
+    - flannel
+    - docker
```

Comment:
We really need to add the kubelet to the master also, since you run fluentd as a DaemonSet. I'll do that in a follow-up. You want to run the kubelet on the master as "Unschedulable" to stop regular workloads from going there.